testing/constraints-3.7.txt

```diff
@@ -0,0 +1 @@
+pandas==1.1.0
```

Comment:
I see that `pandas==1.1.0` still supports Python 3.6, thus presumably this constraint is to test dtypes-related things that we use here and are not necessarily available in less recent pandas version, forcing to use our own forward ported versions?

It would probably be good to explain in a comment what feature differences do the cutoff points 1.1.0 and 1.2.0 (below) represent.

Reply:
https://github.com/googleapis/python-db-dtypes-pandas/pull/9/commits/57cfc2e9c487a71a1f09482f07f9f7701c85f282